src/main/java/org/mastodon/mamut/collaboration/dialogs/SetAuthorDialog.java

```diff
@@ -0,0 +1,42 @@
+package org.mastodon.mamut.collaboration.dialogs;
+
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+import net.miginfocom.swing.MigLayout;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+public class SetAuthorDialog
+{
+	private SetAuthorDialog()
+	{
+		// prevent from instantiation
+	}
+
+	public static Pair< String, String > show()
```

Comment:
I admit, this method is quite self-explaining. But maybe, you could still add some basic javadoc here, since the two returned values cannot be just guessed from the method name.